package.json

```diff
@@ -31,6 +32,7 @@
     "react-native": "0.70.8",
     "react-native-dotenv": "^3.4.0",
     "react-native-error-boundary": "^1.1.16",
+    "react-native-get-random-values": "^1.11.0",
```

Comment:
Required for the compatibility layer, but wc is derp and doesn't include it so need to install separately

Reply:
Actually not required.